client/components/FailingAssertionsSummary/ReportAtBugModal.jsx

```diff
@@ -0,0 +1,113 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import BasicModal from '../common/BasicModal';
+import AssertionDetails from './BugLinking/AssertionDetails';
+
+const ReportATBugModal = ({
+  show,
+  onClose,
+  assertion,
+  testPlanReport,
+  testPlanVersion
+}) => {
+  const handleReportBug = () => {
+    const vendorIssuesUrl = 'https://github.com/w3c/aria-at';
```

Comment:
To avoid unintentionally creating issues on the main repo, you could route these to bocoup repos. Similar to what's happening in [createIssueLink.js](https://github.com/w3c/aria-at-app/blob/bb83bc1e139abe859948397a3b78ca0be46d20eb/client/utils/createIssueLink.js#L1:L4)